ai/core/tests/core/databricks/test_databricks_integration_tests.py

```diff
@@ -88,34 +81,6 @@ def test_create_and_execute_function(
             assert result.value == function_sample.output
 
 
-@retry_flaky_test()
-@requires_databricks
-def test_create_and_execute_retriever_function(serverless_client: DatabricksFunctionClient):
```

Comment:
This test is redundant with the library integration tests that are testing actual e2e tracing capabilities for each integration library. Removing this from here.